src/components/Compendia/NormalizedTab.js

```diff
@@ -38,7 +37,7 @@ export const NormalizedTab = ({ type = 'normalized' }) => {
             quantile normalization.
           </Heading>
           <Button
-            aria-label={`Go to the refinebio docs - ${tabName}`}
+            aria-label="Go to the refinebio docs - Normalized Compendia"
```

Comment:
Since the `aria-lable` value is static (specific to the nomalized tab), I've removed `getReadable` and hard-coded the value here.